service-api/docker/app/Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN adduser -D -g '' appuser
 RUN apk add --upgrade apk-tools
 RUN apk upgrade
 
-RUN apk add --no-cache postgresql-libs icu \
+RUN apk add --no-cache postgresql-libs icu fcgi \
```

Comment:
Could we use wget to make the requests? Just wondering why we need fcgi for this.

Reply:
PHP-FPM will issue a TCP connection reset because wget will try using HTTP instead of the FastCGI protocol. fcgi will exit cleanly (i.e. with exit code 0) if it gets a valid FastCGI response. wget / curl will always exit uncleanly because of the TCP reset.